litestar/contrib/sqlalchemy/repository/_async.py

```diff
@@ -219,11 +238,18 @@ async def exists(self, **kwargs: Any) -> bool:
         existing = await self.count(**kwargs)
         return existing > 0
 
+    def _to_lambda_stmt(
```

Comment:
```suggestion
    def _get_base_stmt(
```

would be more appropriate I think. The lambda statement conversion is more of an implementation detail and the current name hides the fact that this also optionally returns the class instance base statement.